lib/body.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * checks if the description is empty
+ *
+ * @return
+ *  JSON object with the properties mergeable and the description if
+ *  `mergeable` is false
+ */
+module.exports = async (pr, context, settings) => {
+  let body = pr.body
+  let bodySettings = settings.mergeable.body
+  let isMergeable = true
+
+  if (bodySettings === 'non-empty') {
```

Comment:
We shouldn't need this conditional. Handler will not call this validator when the user specifies exclusions in the YML:

`excludes: 'body'` 

We don't need settings at all. `Body` validation is still by default validating for zero length body.